library/src/main/java/com/mapzen/android/DI.java

```diff
@@ -1,4 +1,4 @@
-package com.mapzen.android.dagger;
+package com.mapzen.android;
```

Comment:
I like this, I think it makes sense to do and will definitely clean things up nicely for us. I would consider `MapzenMap` a `com.mapzen.android.graphics` class though, especially because it relies on some package private classes that will squarely reside in `com.mapzen.android.graphics` (OverlayManager, MapStateManager). What do you think?